layouts/partials/footer/footer.html

```diff
@@ -13,7 +13,13 @@
     {{ $.Scratch.Set "data" $dot.Page.Site.Data.partials.footer }}
 {{ end }}
 
+<!--- footer from websites-modules --->
 {{ partial "footer.html" . }}
 
 <!--- global modals --->
-{{ partial "global-modals/global-modals.html" . }}
+{{ partialCached "global-modals/global-modals.html" . }}
```

Comment:
This seemed fine on first inspection but looking closer won't this cause the i18n calls in the html to be cached? we might end up with the ja output used for all or the en output used for all etc.

Reply:
I would be opposed to moving the modal to the baseof instead. I figured the footer/footer was originally cached because global-modals didnt have any conditionals included in the partial

Reply:
i moved it to baseof and cached the footer again